### ISSUE TITLE
[2.0] Repl Window hangs on history navigation

### ISSUE THREAD

**Issue by PEZ** [BetterThanTomorrow/calva#177]
I have a project in which I can not go up in history beyond a certain slot. When I reach that slot and try to go up one step more, this exception is thrown:

```
Uncaught TypeError: Failed to execute 'removeChild' on 'Node': parameter 1 is not of type 'Node'.
    at ReplReadline.repaint (readline.js:511)
    at Object.history-up (repl-console.js:314)
    at HotKeyTable.execute (hotkeys.js:114)
    at HTMLInputElement.ReplConsole.input.addEventListener.capture (repl-console.js:367)
```

The REPL Window is after this totally unresponsive. And exceptions like that are thrown for anything I try to do in the window.

The form in the particular slot I can't go beyond has an unbalanced closing paren. (I guess this has nothing to do with anything.)

I fail to reproduce it in the `repl-indicator`.

**Comment by PEZ** [comment]
I can't figure this one out myself. Now I added a command to clear the REPL Window history, since this condition freezes up the window when it happens. So that can be the workaround for the user.

Can you have a look at this @mseddon ? For me the error happened when, in `readline.ts`, the models deleted lines are processed. The error thrown was about the `removeChild` failing because the item was not a `Node`. (And now I can't reproduce it since I've cleared my history...).

**Comment by sergeyklay** [comment]
Just faced with the same bug

**Comment by cfehse** [comment]
@PEZ  @sergeyklay 

This is fixed in version 2.0.55.
